dev.db.persistent.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Bash strict
+set -euo pipefail
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd "$SCRIPT_DIR" # cd to this script's dir
+
+set -a && source .env.local && set +a
+
+# docker container with database stored on host OS filesystem so it persists
+docker run --rm \
+  -d \
+  --name postgresql-container \
+  -p 5432:5432 \
+  -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD \
+  -v "$SCRIPT_DIR/dev.db.perstence.volume":/var/lib/postgresql/data \
+  postgres
+
+#### Update the datastore schema
+yarn run db:update
+
+#### Seed the datastore
+yarn db:init
+
+echo Database is now running the background. Use ./dev.db.stop.sh to stop it.
```

Comment:
Love these scripts! Please either update the `DEVELOPMENT.md` documentation in this PR, or just remind me to test & update the docs after this lands.